comps/comps-foreman-fedora21.xml

```diff
@@ -38,6 +38,8 @@
       <packagereq type="default">rubygem-autoparse</packagereq>
       <packagereq type="default">rubygem-bootstrap-sass</packagereq>
       <packagereq type="default">rubygem-clamp</packagereq>
+      <packagereq type="default">rubygem-concurrent-ruby</packagereq>
+      <packagereq type="default">rubygem-concurrent-ruby-edge</packagereq>
```

Comment:
-edge should stay in plugins I think, it's a separate package and isn't used here.